controls/V-71993.rb

```diff
@@ -62,8 +42,9 @@
 
 [org/gnome/settings-daemon/plugins/media-keys]
 logout=’’"
+  tag "fix_id": "F-78345r2_fix"
 
   describe systemd_service('ctrl-alt-del.service') do
-    it {should_not be_running }
+    it { should_not be_running }
```

Comment:
This is a `target`, not a `service` and should probably be checked as `should_not be_enabled`.

Also, there should be a `masked?` method added to the `systemd_service` resource since that is a very specific state.